.gitignore

```diff
@@ -2,4 +2,7 @@ packages
 .vs/
 *.suo
 */bin/*
-*/obj/*
\ No newline at end of file
+*/obj/*
+TicketsDemo/App_Data/Trains.txt
```

Comment:
you've ditignored txt files, instead of csv files and therefore (i guess) some csv files are still commited and pushed, such as
TicketsDemo/App_Data/Trains.csv